cg/cli/upload/vogue.py

```diff
@@ -150,6 +157,41 @@ def bioinfo(context, case_name, cleanup, target_load, dry):
             context.obj["vogue_upload_api"].load_bioinfo_sample(load_bioinfo_raw_inputs)
 
 
+@vogue.command("bioinfo-all", short_help="Load all mip bioinfo results into vogue")
```

Comment:
Is it only mip who produce bioinfo results :)
```suggestion
@vogue.command("bioinfo-all", short_help="Load all bioinfo results into vogue")
```

Reply:
For now yes 😉 . Microsalt is next!

Reply:
What about balsamic?

Reply:
balsamic doesn't have any data stored in HK yet. I suppose it'll be loaded when it is stored.

Reply:
review addressed. Microsalt will be another PR if you agree.